e2e-tests/runtimeTests.spec.ts

```diff
@@ -81,7 +82,7 @@ describe('Runtime Tests for `/runtime/*`', () => {
 	/**
 	 * Allows a timeout of 30 seconds for each response.
```

Comment:
remove these comments or modify the timeout? :)

```suggestion
```

Reply:
Thanks for reminding me totally forgot to remove the comments!

Reply:
fixed